nav_msgs/OccupancyGrid map1
nav_msgs/OccupancyGrid map2
bool crop   # Should the output be cropped to show only the region of interest?
---
#geometry_msgs/Transform ros_transform         # Standard ROS transformation
mrgs_alignment/CTransform mapmerge_transform  # Non-standard transformation, direct output from algorithm
nav_msgs/OccupancyGrid merged_map
